examples/protobuf/services.pb.go

```diff
@@ -0,0 +1,1992 @@
+// Copyright 2021 The Kubernetes Authors.
```

Comment:
we should not check-in `examples` dir. This is only to verify the diff with the generated one vs the checked-in one.
Also, I think a better name to this dir would be something like `verify` or `validate`

Reply:
Where should I keep the verify dir ?

Reply:
hack/verify... is fine @yashsingh74